pkg/kube/inject/webhook_test.go

```diff
@@ -69,57 +67,6 @@ imagePullSecrets:
 `,
 }
 
-var (
```

Comment:
Since the rotated cert test is removed - I assume there is a duplicate test checking the same thing ? We had big issues with cert rotation in the past - in particular if we start doing hourly rotation for istiod own cert as we do for workloads. Most of the issues only visible in the stability tests - so I don't mind removing the unit test if we can confirm stability tests can run with hourly (or faster) rotation for istiod certs.